packages/components/atoms/f-form-field/src/components/FormLabel.vue

```diff
@@ -6,7 +6,6 @@
         :class="[
             $style['o-form-label'],
             $style['c-formField-label'], {
-                [$style['c-formField-label--inline']]: isInline,
```

Comment:
You can delete the classes to do with this now – like `.c-formField-label--inline` and the associated selectors with this class in it.